count-lines-of-code

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-cloc --exclude-list-file <(echo -e 'kitty/wcwidth-std.h\nkitty/glfw.c\nkitty/keys.h\nkitty/charsets.c\nkitty/unicode-data.c\nkitty/key_encoding.py\nkitty/rgb.py\nkitty/gl.h\nkitty/gl-wrapper.h\nkitty/gl-wrapper.c\nkitty/glfw-wrapper.h\nkitty/glfw-wrapper.c\nkitty/emoji.h\nkittens/unicode_input/names.h') kitty kittens
+cloc --exclude-list-file <(echo -e 'kitty/wcwidth-std.h\nkitty/glfw.c\nkitty/keys.h\nkitty/charsets.c\nkitty/unicode-data.c\nkitty/key_encoding.py\nkitty/rgb.py\nkitty/gl.h\nkitty/gl-wrapper.h\nkitty/gl-wrapper.c\nkitty/glfw-wrapper.h\nkitty/glfw-wrapper.c\nkitty/emoji.h\nkittens/unicode_input/names.h\nkitty/parse-graphics-command.h') kitty kittens
```

Comment:
We could refactor this like that:
```sh
cloc_ignored_files=(
    kitty/wcwidth-std.h
    kitty/glfw.c
    kitty/keys.h
    kitty/charsets.c
    kitty/unicode-data.c
    kitty/key_encoding.py
    kitty/rgb.py
    kitty/gl.h
    kitty/gl-wrapper.h
    kitty/gl-wrapper.c
    kitty/glfw-wrapper.h
    kitty/glfw-wrapper.c
    kitty/emoji.h
    kittens/unicode_input/names.h
)

cloc --exclude-list-file <(IFS=$'\n'; echo "${cloc_ignored_files[*]}") kitty kittens
```
(I'm not on a linux computer right now, so can't test, feel free to make a PR, or I'll do it later)

Reply:
Except for the missing line I added in this PR, this works as intended for me. I'll let you make the PR.